src/rust/ensogl/lib/components/src/selector/shape.rs

```diff
@@ -16,6 +16,7 @@ use crate::shadow;
 /// Utility struct that contains the background shape for the selector components, as well as some
 /// meta information about it. This information can be used to align other shapes with the
 /// background.
+#[allow(dead_code)]
```

Comment:
I would put here (and in other dead_code places) an explanation what is dead and why. Perhaps @wdanilo should provide it.

Reply:
I've added the notes. (This is my dead code)